lib/plugins/plugins.rb

```diff
@@ -32,8 +32,10 @@ def load
     file_location = @settings.plugins['location'] ? @settings.plugins['location'] : '../../config/plugins/'
 
     yaml_files = File.expand_path(file_location + '*.yml', File.dirname(__FILE__))
+    port = 48130
```

Comment:
We have to provide a starting point to iterate from, you'll notice there is a `port += 1` below this.

That way every plugin loaded is assigned a unique port (e.g.; 48130, 48131, 48132)